tests/Decoder.js

```diff
@@ -188,6 +200,15 @@ test('Decode nested tuple arguments', t => {
     )
 })
 
+test('Decode long tuple arguments', t => {
+    t.plan(1)
+    t.deepEqual(
+        encoder.decode(CONTRACT, 'test_long_tuple', 'cb_CwFUVFRUVFRUVFRUVFRUVFRUVPz3ok8='),
```

Comment:
You can also get this value in #149, but cli can't accept long tuples currently, so I changed the contract:
```
entrypoint test_long_tuple(): int * int * int * int * int * int * int * int * int * int * int * int * int * int * int * int * int = (42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42, 42)
```
```
./tests/contract-call.sh test_long_tuple "[]"

File not found
test_long_tuple([]) -> cb_CwFUVFRUVFRUVFRUVFRUVFRUVPz3ok8=
```

Reply:
okay, did you tracked an issue about supporting it ?

Reply:
firstly I need calldata released, then I will release sdk depending on it and after that will update and release cli, everything should work fine after that